src/reason-parser/reason_attributes.ml

```diff
@@ -38,6 +38,9 @@ let rec partitionAttributes ?(partDoc=false) ?(allowUncurry=true) attrs : attrib
   | (({txt="reason.raw_literal"}, _) as attr) :: atTl ->
     let partition = partitionAttributes ~partDoc ~allowUncurry atTl in
     {partition with literalAttrs=attr::partition.literalAttrs}
+  | (({txt="reason.preserve_braces"}, _) as attr) :: atTl ->
+    let partition = partitionAttributes ~partDoc ~allowUncurry atTl in
+    {partition with literalAttrs=attr::partition.literalAttrs}
```

Comment:
`literalAttrs` was originally intended to represent string literals. It seems you're repurposing them here? The name should probably be changed accordingly.